openff/fragmenter/tests/test_fragmenter.py

```diff
@@ -417,13 +441,17 @@ def test_prepare_molecule():
 )
 def test_fragmenter_provenance(toolkit_registry, expected_provenance):
```

Comment:
Sorry if I have missed this, but it would be good to add the `rotatable_bond_smarts` string used to the provenance info. So this should either be the default or the string provided by the user I think.

Reply:
I forgot to say it would also be good to test for it and make sure it was added correctly :D

Reply:
Good call - added in b4ec545